packages/fscomponents/src/components/Grid.tsx

```diff
@@ -484,6 +484,7 @@ export const Grid = <ItemT, >(props: GridProps<ItemT>) => {
         onEndReachedThreshold={onEndReachedThreshold}
         onLayout={onLayout}
         onContentSizeChange={setWidth}
+        extraData={[...listViewProps?.extraData, renderRow]}
```

Comment:
If extraData is undefined this is going to error when we try to spread on it. Should we add a check for that?

Reply:
Oh! extraData is `any` so I wasn't getting errors for that.
Let me fix this.